http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyRequestExecutor.java

```diff
@@ -162,6 +165,7 @@ private boolean tryConfigurePipeline() {
             return false;
         }
 
+        pipeline.addLast(LastHttpContentHandler.create());
```

Comment:
Did we verify this works for HTTP/2?

Reply:
Verified with h2 test service. I'll double check with the actual h2 service.

Reply:
Verified it works. 

Also checked the code and we do convert h2 endstream to a `DefaultLastHttpContent`.

https://github.com/aws/aws-sdk-java-v2/blob/4c0e42ed66975de42d68df9d508ffd9584d4428f/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/http2/Http2ToHttpInboundAdapter.java#L63-L69